### PR TITLE
Mage_Core - DOC block update

### PR DESCRIPTION
**Pull request by sreichel**
#692 just from own repo.

### CODE REVIEW

File: app/code/core/Mage/Core/Model/Translate.php [hunk 1]
```diff
@@ -412,7 +424,16 @@ public function translate($args)
 
         //array_unshift($args, $translated);
         //$result = @call_user_func_array('sprintf', $args);
-
+        if (substr_count($translated, '%') != count($args)) {
```
Comment (kkrieger85) [inline app/code/core/Mage/Core/Model/Translate.php]: Code change. Could you please separate this to an extra PR?

Comment (sreichel) [reply app/code/core/Mage/Core/Model/Translate.php]: Sorry, This doesnt belong in at all.... was just to debug some translation. I'll remove ...

File: app/code/core/Mage/Core/Block/Abstract.php [hunk 2]
```diff
@@ -942,10 +941,12 @@ public function setFrameTags($openTag, $closeTag = null)
         if (self::$_transportObject === null) {
             self::$_transportObject = new Varien_Object;
         }
-        self::$_transportObject->setHtml($html);
-        Mage::dispatchEvent('core_block_abstract_to_html_after',
-            array('block' => $this, 'transport' => self::$_transportObject));
-        $html = self::$_transportObject->getHtml();
+        self::$_transportObject->setData('html', $html);
```
[the remark this answers is not in the document]
Comment (sreichel) [reply app/code/core/Mage/Core/Block/Abstract.php]: Left over from 1st PR ... will be reverted to.

File: app/code/core/Mage/Core/Model/App/Emulation.php [hunk 3]
```diff
@@ -100,7 +102,7 @@ public function startEnvironmentEmulation($storeId, $area = Mage_Core_Model_App_
      *
      * Function restores initial store environment
      *
-     * @param Varien_Object $initialEnvironmentInfo information about environment of the initial store
+     * @param Varien_Object|OpenMage_Object_AppEmulation $initialEnvironmentInfo information about environment of the initial store
```
[the remark this answers is not in the document]
Comment (sreichel) [reply app/code/core/Mage/Core/Model/App/Emulation.php]: There isn't a PR yet ... left over from tests (https://github.com/OpenMage/magento-lts/issues/687#issuecomment-498442813) ... i'll remove.

File: app/code/core/Mage/Core/Model/Observer.php [hunk 4]
```diff
@@ -110,12 +109,12 @@ public function cleanCache(Mage_Cron_Model_Schedule $schedule)
     /**
      * Cleans cache by tags
      *
-     * @param Varien_Event_Observer $observer
+     * @param Varien_Event_Observer|OpenMage_Observer_CleanCacheByTags_Observer $observer
```
[the remark this answers is not in the document]
Comment (sreichel) [reply app/code/core/Mage/Core/Model/Observer.php]: Shouldnt be in ... see #687.

Thanks for reviewing. :)

File: app/code/core/Mage/Core/Model/Resource.php [hunk 5]
```diff
@@ -50,7 +49,7 @@ class Mage_Core_Model_Resource
     /**
      * Instances of actual connections
      *
-     * @var array
+     * @var Varien_Db_Adapter_Interface|false[]
```
Comment (tmotyl) [inline app/code/core/Mage/Core/Model/Resource.php:52]: Varien_Db_Adapter_Interface[] ?

File: app/code/core/Mage/Core/Model/Resource.php [hunk 6]
```diff
@@ -127,7 +127,7 @@ public function getConnection($name)
     /**
      * Get Instances of actual connections
      *
-     * @return array
+     * @return false[]|Varien_Db_Adapter_Interface
```
Comment (tmotyl) [inline app/code/core/Mage/Core/Model/Resource.php:130]: Varien_Db_Adapter_Interface[]

**Pull request merged**